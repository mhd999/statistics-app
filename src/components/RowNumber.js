// @flow

import React from 'react';
import styler from 'react-styling';

type Props = {
    number: number,
};

const style = styler `
  TextStyle
    font-size: 12px
    color: #000000
 `;
const RowNumber = (props: Props) => (
  <span style={style.TextStyle}>{props.number}</span>
);

export default RowNumber;