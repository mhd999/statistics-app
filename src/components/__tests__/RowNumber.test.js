import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import RowNumber from '../RowNumber';

describe('<RowNumber />', () => {
  it('renders a number', () => {
    const component = shallow(
      <RowNumber number={123} />
    );
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
