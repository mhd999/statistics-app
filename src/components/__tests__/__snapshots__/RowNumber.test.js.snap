exports[`<RowNumber /> renders a number 1`] = `
<span
  style={
    Object {
      "color": "#000000",
      "fontSize": "12px",
    }
  }>
  123
</span>
`;
